client/ayon_houdini/plugins/create/create_bgeo.py

```diff
@@ -60,10 +66,17 @@ def create(self, product_name, instance_data, pre_create_data):
         instance_node.setParms(parms)
 
     def get_instance_attr_defs(self):
+        render_target_items = {
+            "local": "Local machine rendering",
+            "local_no_render": "Use existing frames (local)",
+            "farm": "Farm Rendering",
+            "farm_no_render": "Use existing frames (farm)"
```

Comment:
I'm pretty sure we'll need matching PRs on `ayon-deadline` to support this. @MustafaJafar thoughts?

I do like the feature though.